### ISSUE TITLE
REFACTOR - vf-inlay breakpoint should use $vf-breakpoint--md

### ISSUE THREAD

**Issue by khawkins98** [visual-framework/vf-core#757]
Currently vf-inlay can cause "odd" effects on content between the large (desktop, 1024px) and medium (tablet, 768px) breakpoints. 

When the screen size is between the two breakpoints (say, 900px) that `vf-inlay__content--main` gets bigger than it is on desktop. 

What I'm thinking is that vf-inlay should instead break on the medium breakpoint, so `vf-inlay__content--main` won't become bigger on tablet than it is on desktop.

I made this diagram to help demonstrate: 

![Responsive vf-inlay](https://user-images.githubusercontent.com/928100/74126617-aadf4700-4bd8-11ea-9944-ddf2c421a929.png)

[Google docs source](https://docs.google.com/drawings/d/1d_J00I95p7Iue08vnqBBbAzIrurEqF64cE3BeDM6sNc/edit?usp=sharing)

**Comment by sturobson** [comment]
if I recall correctly, the vf-inlay grid has similarities to an existing grid (embl-grid maybe) so, there's a high chance the CSS is/was the same. Perhaps there's something in layouts having their own set of breakpoints (or we could also look at making use of CSS grids 'built-in' ways to help us to do this without making use of media queries at all).

I'm going to go out on a limb and say at some point the 'right hand column' is going to have content that looks "odd" when it's only 250px wide.

Perhaps we need to step away from 'generalised device breakpoints' completely

```
$vf-breakpoint--xs: 360px;
$vf-breakpoint--sm: 600px;
$vf-breakpoint--md: 768px;
$vf-breakpoint--lg: 1024px;
$vf-breakpoint--xl: 1280px;
```

to become 

```
$vf-breakpoint--xs: 400px;
$vf-breakpoint--sm: 600px;
$vf-breakpoint--md: 800px;
$vf-breakpoint--lg: 1000px;
$vf-breakpoint--xl: 1200px;
```

**Comment by sturobson** [comment]
I'm also wondering, as an aside but probably from where this issue comes from - does embl.org/news _need_ to use `vf-inlay` at all.

**Comment by khawkins98** [comment]
On a large desktop screen the `vf-inlay__content--main` has a max width of 798px, but on tablet that `--main` jumps to 959px in width, which makes images rather large and the text line length too long. 

I can think of two ways to solve that:

1. In `vf-inlay` drop the media query to medium/768px which makes the content behave more "normally"
2. "float" the `--main` when between 768px and 1024px, which might be nice but would add a lot more code/edge case stuff

Agree about News being a bit of a free radical as we've not had time to asses components/patterns for it, but this is an issue that also shows up on the group blog posts: https://dev.beta.embl.org/groups/dummy-embl-group/blog/2020/02/test-post/

Here's a quick codepen of how that first fix would work at it's simplest by using the medium breakpoint: https://codepen.io/khawkins98/pen/mdJJJeX?editors=1100

Basically it trades off the risk of very large images/long lines by making the `--main` perform as small mobile layout in the awkward spot. 

It also still gets whatever code has been inherited/reused from embl-grid, probably that's why it's also keeping the sidebar at a minimum 22em.

---

> Perhaps we need to step away from 'generalised device breakpoints' completely

👍

**Comment by sturobson** [comment]
we've kinda made `vf-inlay` redundant now - can we close this?

**Comment by khawkins98** [comment]
👍
